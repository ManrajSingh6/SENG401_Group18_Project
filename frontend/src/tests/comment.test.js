import React from 'react';
import renderer from 'react-test-renderer';
import ThumbUpIcon from '@mui/icons-material/ThumbUpOffAlt';
import ThumbDownIcon from '@mui/icons-material/ThumbDownOffAlt';
import Dropdown from '../components/comment';


//testing like comment
test('renders thumbUpIcon', () => {
  const testme = renderer.create(<ThumbUpIcon />).toJSON();
  expect(testme).toMatchSnapshot();
});

// testing dislike comment
test('renders thumbDownIcon', () => {
    const testme = renderer.create(<ThumbDownIcon />).toJSON();
    expect(testme).toMatchSnapshot();
  });