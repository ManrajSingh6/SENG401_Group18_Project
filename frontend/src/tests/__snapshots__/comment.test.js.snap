// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders thumbDownIcon 1`] = `
<svg
  aria-hidden={true}
  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
  data-testid="ThumbDownOffAltIcon"
  focusable="false"
  viewBox="0 0 24 24"
>
  <path
    d="m10.89 18.28.57-2.89c.12-.59-.04-1.2-.42-1.66-.38-.46-.94-.73-1.54-.73H4v-1.08L6.57 6h8.09c.18 0 .34.16.34.34v7.84l-4.11 4.1M10 22l6.41-6.41c.38-.38.59-.89.59-1.42V6.34C17 5.05 15.95 4 14.66 4h-8.1c-.71 0-1.36.37-1.72.97l-2.67 6.15c-.11.25-.17.52-.17.8V13c0 1.1.9 2 2 2h5.5l-.92 4.65c-.05.22-.02.46.08.66.23.45.52.86.88 1.22L10 22zm10-7h2V4h-2c-.55 0-1 .45-1 1v9c0 .55.45 1 1 1z"
  />
</svg>
`;

exports[`renders thumbUpIcon 1`] = `
<svg
  aria-hidden={true}
  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
  data-testid="ThumbUpOffAltIcon"
  focusable="false"
  viewBox="0 0 24 24"
>
  <path
    d="m13.11 5.72-.57 2.89c-.12.59.04 1.2.42 1.66.38.46.94.73 1.54.73H20v1.08L17.43 18H9.34c-.18 0-.34-.16-.34-.34V9.82l4.11-4.1M14 2 7.59 8.41C7.21 8.79 7 9.3 7 9.83v7.83C7 18.95 8.05 20 9.34 20h8.1c.71 0 1.36-.37 1.72-.97l2.67-6.15c.11-.25.17-.52.17-.8V11c0-1.1-.9-2-2-2h-5.5l.92-4.65c.05-.22.02-.46-.08-.66-.23-.45-.52-.86-.88-1.22L14 2zM4 9H2v11h2c.55 0 1-.45 1-1v-9c0-.55-.45-1-1-1z"
  />
</svg>
`;
